indexing-hadoop/src/main/java/io/druid/indexer/IndexGeneratorJob.java

```diff
@@ -391,8 +557,14 @@ protected void reduce(
             persist(index, interval, file, progressIndicator);
             // close this index and make a new one, reusing same buffer
             index.close();
-            index = makeIncrementalIndex(bucket, aggs, bufferPool);
 
+            index = makeIncrementalIndex(
```

Comment:
it is closed at line 559 (after canAppendRow check) and 647 (in the finally block)